src/main/scala/com/github/catalystcode/fortis/spark/streaming/reddit/client/RedditClient.scala

```diff
@@ -0,0 +1,197 @@
+package com.github.catalystcode.fortis.spark.streaming.reddit.client
+
+import java.net.URLEncoder
+import java.util.UUID
+
+import com.github.catalystcode.fortis.spark.streaming.reddit.Logger
+import com.github.catalystcode.fortis.spark.streaming.reddit.dto.RedditObject
+import net.liftweb.json
+
+import scalaj.http._
+
+/**
+  * The RedditClient class provides two public-facing methods:
+  * <ul>
+  *   <li>{@link #ensureFreshToken()}</li>
```

Comment:
Another example of a comment that in my opinion is redundant since the class structure already tells us that there are two public methods ;) I really like this [article](https://ncrcoe.gitbooks.io/java-for-small-teams/style/20_prefer_readable_code_to_comments.html) for comment philosophy.